sdk/provisioning/Azure.Provisioning/tests/Infrastructure/AppConfiguration/main.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
I would create on the fly and delete these.  Is a testing artifact only which doesn't take that long to create.

Reply:
done